src/GitExtensions.GerritPlugin/FormGitReview.cs

```diff
@@ -26,6 +26,16 @@ public sealed partial class FormGitReview : GitExtensionsForm, IGitUICommandsSou
         private readonly TranslationString _saveFileQuestionCaption =
             new TranslationString("Save changes?");
 
+        private readonly TranslationString _description = new TranslationString(@$"Example configuration
+
+                                                                                   [gerrit]
+                                                                                   host=review.example.com
+                                                                                   port=29418
+                                                                                   project=department/project.git
+                                                                                   defaultbranch=master
+                                                                                   defaultremote=review
+                                                                                   defaultrebase=0");
```

Comment:
Just make sure these leading spaces won't be a problem
```suggestion
        private readonly TranslationString _description = new TranslationString(@$"Example configuration

[gerrit]
host=review.example.com
port=29418
project=department/project.git
defaultbranch=master
defaultremote=review
defaultrebase=0");
```